pkg/tuned/controller.go

```diff
@@ -80,7 +80,7 @@ const (
 	// be generous and give it 10s.
 	tunedGracefulExitWait = time.Second * time.Duration(10)
 	ocpTunedHome          = "/var/lib/ocp-tuned"
-	ocpTunedRunDir        = "/run/ocp-tuned"
+	ocpTunedRunDir        = "/run/" + programName
```

Comment:
intentionally reverted?

Reply:
It is actually not a "functional" revert, because of 
```
OperandFilename          = "ocp-tuned"
```
in `version.go`
so yes, this is intentional.